scripts/local-docker-integration-tests.sh

```diff
@@ -11,25 +11,29 @@ REPO_ROOT=`dirname "$0"`/..
 DOCKER_COMPOSE_FILE=$REPO_ROOT/docker/docker-compose.ci.yml
 EXTRA_ARGS=$*
 
-# environment variables passed in. Update as needed for testing
-export DB_USERNAME='username'
-export DB_PASSWORD='password'
-export DB_NAME='locksmith'
-export DB_HOSTNAME='db'
-export CI=true
-export HTTP_PROVIDER='ganache-integration'
-export LOCKSMITH_URI='http://locksmith:8080'
-export PAYWALL_URL='http://unlock:3001'
-export PAYWALL_SCRIPT_URL='http://unlock:3001/static/paywall.min.js'
-export UNLOCK_STATIC_URL='http://unlock-protocol-com:3002'
+# set the environment variables needed for integration testing
+eval "$($REPO_ROOT/scripts/set-integration-tests-env-variables.sh)"
 
 # if the integration test images are running, this ensures we remove any state
 # prior to attempting to run the tests. This line is critical!
 docker-compose -f $DOCKER_COMPOSE_FILE down
 
 # re-build the images. This will use local docker cache
-docker build -t unlock -f "$REPO_ROOT/docker/unlock.dockerfile" $REPO_ROOT
-docker build -t unlock-integration -f "$REPO_ROOT/docker/integration-tests.dockerfile" $REPO_ROOT
+# and because we source the script, it will inherit our environment variables
```

Comment:
bad news: this is the only method by which the 2nd build uses the cached builds. To make the docker-compose-build script work for local runs will require extensive re-writing. With the below script, the 2nd (and consequent) runs of integration tests takes only 45 seconds to rebuild